### DISCUSSION TITLE
project

### DISCUSSION THREAD

**Discussion by jbzon** [OpenMined/courses#133] [Project: Privacy Product Specification] [answered]
i didnot understand what exactly i have to do in this project?

**Comment by yemikifouly** [comment]
Hello! Welcome to the discussion board! 🙂 Have you completed the course already?

**Reply by jbzon** [reply]
yes

**Reply by yemikifouly** [reply]
Let me get you help. I think @mcleonard will be able to help you. 🙂

**Comment by jbzon** [comment]
yes i have

**Comment by jbzon** [comment]
i didnot understand what exactly i have to do in this project? @mcleonard

**Comment by jbzon** [comment]
can you tell me how to communicate @mcleonard .I am not able to navigate?

**Comment by mcleonard** [comment, marked as answer]
Hi there @jbzon! For the project you should write a document that describes a product or service that uses privacy technology. You can write it directly in the classroom, or some other text editor then copy & paste it into the classroom.

You'll need to specify what the product/service does, how people use it, the value proposition (why people would want to use it), and other things listed in the project description.

Do you have any specific questions?

**Reply by jbzon** [reply]
I mean I wrote quite a lot and in the classroom i guess there is a word limit. So is it compulsory to shorten your product specification details i I wonder?

**Reply by jbzon** [reply]
I submitted my project but it was not complete since there was a word limit. Do I need to remove the details in order to fit in with the word limit in my second and third attempt?

**Reply by mcleonard** [reply]
Oh a word limit? That might be something we need to fix. How many words did you have?

**Reply by jbzon** [reply]
169 words

**Reply by mcleonard** [reply]
Something went wrong somewhere, you should be able to input 4-5 pages of text. I'll check it out today.

**Reply by jbzon** [reply]
Let me know when you are done checking . Since there is a deadline , i assume.

**Reply by mcleonard** [reply]
There's no deadline. No worries about that.

**Reply by mcleonard** [reply]
Hi @jbzon. I submitted a project that is 2 pages long, roughly 820 words. No problems. The project submission should be able to handle up to ~1 MB of data which is ~6 pages. I'm not sure what's going on your side. Maybe you some how copied in a ton of white space characters?

**Reply by jbzon** [reply]
oh! I see .
